content/tutorials/getusermedia/intro/en/index.html

```diff
@@ -98,56 +98,54 @@
 
 <h2 id="toc-into">Introduction</h2>
 
-<p>Audio/Video capture has been <em>the</em> "Holy Grail" of web development for a long time.
-For many years we had to rely on browser plugins (<a href="http://www.kevinmusselman.com/2009/02/access-webcam-with-flash/">Flash</a> or
+<p>The ability to capture audio and video has been <em>the</em> Holy Grail of web development for a long time.
+For many years, you had to rely on browser plugins (<a href="http://www.kevinmusselman.com/2009/02/access-webcam-with-flash/">Flash</a> or
 <a href="https://blogs.msdn.microsoft.com/silverlight_sdk/2010/03/16/using-webcams-in-silverlight-4/">Silverlight</a>)
 to get the job done. <a href="https://www.youtube.com/watch?v=SP_9zH9Q44o" target="_blank">Come on!</a></p>
 <p>HTML5 to the rescue. It might not be apparent, but the rise of HTML5 has brought
 a surge of access to device hardware. <a href="/tutorials/geolocation/trip_meter/">Geolocation</a> (GPS),
 the <a href="/tutorials/device/orientation/">Orientation API</a> (accelerometer), <a href="/tutorials/webgl/shaders/">WebGL</a> (GPU),
 and the <a href="/tutorials/webaudio/intro/">Web Audio API</a> (audio hardware) are perfect examples. These features
-are ridiculously powerful, exposing high level JavaScript APIs that sit
-on top of the system's underlying hardware capabilities.</p>
+are ridiculously powerful, and expose high-level JavaScript APIs that sit
+on top of the system's foundational hardware capabilities.</p>
 <p>This tutorial introduces <a href="https://www.w3.org/TR/mediacapture-streams/"><code>navigator.mediaDevices.getUserMedia()</code></a>, which allows
 web apps to access a user's camera and microphone.</p>
 <h2 id="toc-history">The road to getUserMedia()</h2>
 
-<p>If you're not aware of its history, the way we arrived at the <code>getUserMedia()</code> API is an interesting tale.</p>
-<p>Several variants of "Media Capture APIs" have evolved over the past few years.
-Many folks recognized the need to be able to access native devices on the web, but
-that led everyone and their mom to put together a new spec. Things got
+<p>If you're not aware of its history, the road to the <code>getUserMedia()</code> API is an interesting tale.</p>
+<p>Several variants of media-capture APIs evolved over the past few years.
+Many folks recognized the need to access native devices on the web, but
+that led everyone and their mothers to put together a new spec. Things got
```

Comment:
I think this sounds a bit odd now :). 

Maybe something like 'that led to a lot of different people putting forward new proposals'.

Reply:
Sure, just a slight tweak: "that led many people to put forward new proposals."

Sound good?